rescript/tests/test_orient.py

```diff
@@ -66,6 +66,21 @@ def test_reorient_default(self):
         exp_unmatched_ids = {'JUNK', 'MOREJUNK'}
         self.assertEqual(unmatched_ids, exp_unmatched_ids)
 
+    def test_reorient_no_ref(self):
```

Comment:
why was this test moved? It does not look like anything changed.

Reply:
I moved it to keep the commented tests in the end, but it was unnecessary. Now I just added decorators to skip the failing tests, so in the future if anything changes they can be executed again by simply removing the decorators.

Reply:
I agree with @nbokulich, any parameters that are not available for `vsearch orient` should be removed, along with adding any potentially useful parameters for the `orient` command outlined in the [manual](https://github.com/torognes/vsearch/releases/download/v2.22.1/vsearch_manual.pdf).

Reply:
@mikerobeson Thank you for the suggestion! I just added the optional parameters available with `orient`.